### PR TITLE
[#156799941] Added web app to test elastichsearch query. added sugges…

### PR DESCRIPTION
**Pull request by gmereu**
…tion query code

### CODE REVIEW

File: elasticsearch/html/assets/js/example.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    function(body){
+      $('#es-automplete-results').text("");
+      var names = body.suggest.names.pop();
+      $.each(names.options, function(index, option){
+        $('#es-automplete-results').append("<div>" + option._source.name + "</div>" );
+      });
+    },
+    function(error){console.log(error);}
+  );
+}
```
Comment (lussoluca) [inline elasticsearch/html/assets/js/example.js]: remember to add an empty line at the end of each file

File: elasticsearch/html/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  <body>
+    <h1 class="">Elastich Search Query test</h1>
+    <div>
+      <h2>Autocomplete</h2>
+      <div><input type="text" id="es-automplete-input"></div>
+      <div id="es-automplete-results"></div>
+    </div>
+  </body>
+</html>
```
Comment (lussoluca) [inline elasticsearch/html/index.html]: remember to add an empty line at the end of each file

File: elasticsearch/html/assets/js/example.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+$( document ).ready(function() {
+  var client = new elasticsearch.Client({
+    host: 'http://elasticsearch.developers.loc',
```
Comment (lussoluca) [inline elasticsearch/html/assets/js/example.js]: maybe we can read this from somewhere?

Comment (gmereu) [reply elasticsearch/html/assets/js/example.js]: You mean to put it in a configuration file?

**Pull request merged**